package.json

```diff
@@ -31,41 +32,43 @@
   },
   "dependencies": {
     "@overture-stack/ego-token-middleware": "^1.1.1",
-    "acorn": "^5.7.4",
     "babel-polyfill": "^6.26.0",
     "cors": "^2.8.4",
     "detect-installed": "^2.0.4",
-    "dot-prop": "^4.2.1",
     "dotenv": "^4.0.0",
     "express": "^4.16.2",
     "express-graphql": "^0.6.11",
     "graphql": "^0.11.7",
     "graphql-compose": "^4.0.1",
     "graphql-compose-mongoose": "^4.1.0",
     "graphql-compose-pagination": "^3.0.1",
-    "kind-of": "^6.0.3",
-    "lodash": "^4.17.10",
-    "minimist": "^1.2.5",
```

Comment:
@caravinci  need to keep `minimist` as a dependency too because it's used: `src/persona-scripts/index.ts`
```
 import * as parseArgs from 'minimist';
```

Reply:
actually in `persona-scripts` so a devDependency if we're being particular?

Reply:
The package is still being installed, just as a dependency of something else, which should keep both up to date.

Reply:
Doesn't resolve the package for this file `src/persona-scripts/index.ts` on running `npm i`.

Reply:
Oddly enough, it installs perfectly on my end. What version of Node + NPM are you using?

![image](https://user-images.githubusercontent.com/2107110/100000778-12ddd300-2d90-11eb-8f37-0d8eb9b23a74.png)

Reply:
older verions ha. node 13.14, npm 6.14.4
![juston](https://user-images.githubusercontent.com/1486054/100001145-aca58000-2d90-11eb-8775-ebdb2b632592.png)

Reply:
aaah!!! look at that npm warning in the beginning. the lockfile version is incompatible.

Reply:
We could just make note of a minimum node version in the readme seen as everybody could be on diff versions

Reply:
The question is, should be require the latest Node? or allow outdated versions?

Reply:
14 is also LTS so that's more than fair I think. I'd suggest just double checking with some of the devs who know Persona better, just in case we know of someone using it in a specific way that we might break. better safe than sorry as they say :)

Reply:
Exactly. 14 actually still uses lockfile v1. I'll update the PR to use that version instead and see if that works with your 13 (and potentially with older versions)... that way we don't have to require a minimum version.

Reply:
I've updated the PR with Node 14. Please kindly, try again? @ciaranschutte

Reply:
Opted for adding minimist back in devDependencies, to allow compatibility with older Node versions.